pkg/cloud/services/networking/securitygroups.go

```diff
@@ -289,6 +289,9 @@ func getAllNodesRules(remoteManagedGroups map[string]string, allNodesSecurityGro
 				rules = append(rules, rc)
 			}
 		} else {
+			if rule.RemoteGroupID == nil && rule.RemoteIPPrefix == nil {
+				return nil, fmt.Errorf("remoteManagedGroups or remoteGroupID or remoteIPPrefix must be set for allNodes security group rules")
```

Comment:
This was confusing to me because:
* `openstackclient` sets a default IP prefix to `0.0.0.0/0` if no remote parameter is set. [Source](https://github.com/openstack/python-openstackclient/blob/5e5b89f906f864f50c5bdc1854d120fde3996c87/openstackclient/network/v2/security_group_rule.py#L54).
* Gophercloud doesn't have a default for that string.
* When creating the Security Group Rule with Gophercloud ([reproducer](https://paste.opendev.org/show/b4nLwLkAIQMY7T5mBZc6/)), the returned object has no Remote IP prefix, however `openstackclient` would show it to `0.0.0.0/0` ([logs](https://paste.opendev.org/show/bOYSD86Saq0VKSFNK8BY/)).

So I think we **can** disabling this rule since apparently there is a default provided by Neutron.